providers/dns/bluecat/bluecat.go

```diff
@@ -60,10 +60,13 @@ type DNSProvider struct {
 }
 
 // NewDNSProvider returns a DNSProvider instance configured for Bluecat DNS.
-// Credentials must be passed in the environment variables: BLUECAT_SERVER_URL, BLUECAT_USER_NAME and BLUECAT_PASSWORD.
-// BLUECAT_SERVER_URL should have the scheme, hostname, and port (if required) of the authoritative Bluecat BAM server.
-// The REST endpoint will be appended.
-// In addition, the Configuration name and external DNS View Name must be passed in BLUECAT_CONFIG_NAME and BLUECAT_DNS_VIEW.
+// Credentials must be passed in the environment variables:
+//  - BLUECAT_SERVER_URL
+// 		It should have the scheme, hostname, and port (if required) of the authoritative Bluecat BAM server.
+// 		The REST endpoint will be appended.
+//  - BLUECAT_USER_NAME and BLUECAT_PASSWORD
+//  - BLUECAT_CONFIG_NAME (the Configuration name)
+//  - BLUECAT_DNS_VIEW (external DNS View Name)
```

Comment:
Lists indented by space is rendered on pkg.go.dev in a single line (this [SO post](https://stackoverflow.com/q/52744468/766783) still applies).

One workaround is to use tabs (this will render a code block, so semantically, thats not ideal):

```suggestion
//	- BLUECAT_SERVER_URL
//	  It should have the scheme, hostname, and port (if required) of the authoritative Bluecat BAM server.
//	  The REST endpoint will be appended.
//	- BLUECAT_USER_NAME and BLUECAT_PASSWORD
//	- BLUECAT_CONFIG_NAME (the Configuration name)
//	- BLUECAT_DNS_VIEW (external DNS View Name)
```

An alternative is to use manual bullets:

```suggestion
// • BLUECAT_SERVER_URL
//
//  It should have the scheme, hostname, and port (if required) of the authoritative Bluecat BAM server.
//  The REST endpoint will be appended.
//
// • BLUECAT_USER_NAME and BLUECAT_PASSWORD
//
// • BLUECAT_CONFIG_NAME (the Configuration name)
//
// • BLUECAT_DNS_VIEW (external DNS View Name)
```